### ISSUE TITLE
Puppeter and Google Cloud Build

### ISSUE THREAD

**Issue by ai** [ai/size-limit#132]
Here is my `cloudbuild.yaml`:

```yml
steps:
  - name: node:12
    entrypoint: yarn
    args: ['install']
  - name: node:12
    entrypoint: yarn
    args: ['test']
images:
  - gcr.io/sitnikru-1547645703906/sitnikru
```

Here is an error:

```
Step #1: Error: Failed to launch chrome!
Step #1: /workspace/node_modules/estimo/temp/chrome/linux-674921/chrome-linux/chrome: error while loading shared libraries: libX11-xcb.so.1: cannot open shared object file: No such file or directory
Step #1: 
Step #1: 
Step #1: TROUBLESHOOTING: https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md
Step #1: 
Step #1:     at onClose (/workspace/node_modules/puppeteer-core/lib/Launcher.js:349:14)
Step #1:     at Interface.<anonymous> (/workspace/node_modules/puppeteer-core/lib/Launcher.js:338:50)
Step #1:     at Interface.emit (events.js:214:15)
Step #1:     at Interface.close (readline.js:403:8)
Step #1:     at Socket.onend (readline.js:180:10)
Step #1:     at Socket.emit (events.js:214:15)
Step #1:     at endReadableNT (_stream_readable.js:1178:12)
Step #1:     at processTicksAndRejections (internal/process/task_queues.js:77:11)
Step #1: error Command failed with exit code 1.
Step #1: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Finished Step #1
ERROR
ERROR: build step 1 "node:12" failed: exit status 1
--------------------------------------------------------------------------------

ERROR: (gcloud.builds.submit) build f966db4d-4556-4c35-9039-7130fc5f7ac6 completed with status "FAILURE"
```

@aslushnikov can you recommend how to fix `cloudbuild.yaml` to fix Size Limit on Google Cloud Build?

**Comment by dimaShin** [comment]
Same issue in gitlab-ci
Size-limit version: 2.1.1
And I'm not using --why

**Comment by nikolay-govorov** [comment]
Hey.

Thank you for what you're doing. I got with this problem in GitLab CI. The problem occurs because `puppeteer` cannot install chrome in the Docker container. Read more in the [puppeteer documentation](https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker).

To fix the problem, you need to explicitly install chrome in docker. There is a [ready image](https://hub.docker.com/r/buildkite/puppeteer/).

The working step for Gitlab CI looks like this:
```yaml
test/size-limit:
  image: buildkite/puppeteer:latest # fix for github.com/ai/size-limit/issues/132
  stage: test
  script: yarn size-limit
```

**Comment by ai** [comment]
Great. I improved error text 0fabe05

**Comment by ai** [comment]
New warning released in 2.1.5
